.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ jobs:
       # https://circleci.com/docs/2.0/configuration-reference/#available-machine-images
       image: ubuntu-2004:202101-01
 
-    working_directory: ~/akka-safer-serializer
```

Comment:
Looks like the only file with occurrences of `git grep -i safer` left is README.md

Reply:
Yes, I already rebased #46